test/e2e/placement_apply_strategy_test.go

```diff
@@ -0,0 +1,294 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package e2e
+
+import (
+	"fmt"
+
+	"github.com/google/go-cmp/cmp"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"go.goms.io/fleet/pkg/controllers/work"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	placementv1beta1 "go.goms.io/fleet/apis/placement/v1beta1"
+)
+
+var _ = Describe("validating CRP when resources exists", Ordered, func() {
+	crpName := fmt.Sprintf(crpNameTemplate, GinkgoParallelProcess())
+	annotationKey := "annotation-key"
+	annotationValue := "annotation-value"
+	workNamespaceName := fmt.Sprintf(workNamespaceNameTemplate, GinkgoParallelProcess())
+
+	BeforeAll(func() {
+		By("creating work resources on hub cluster")
+		createWorkResources()
+	})
+
+	AfterAll(func() {
+		By("deleting created work resources on hub cluster")
+		cleanupWorkResources()
+
+		By("deleting created work resources on member cluster")
+		cleanWorkResourcesOnCluster(allMemberClusters[0])
+	})
+
+	Context("Test a CRP place objects successfully (client-side-apply)", Ordered, func() {
+		BeforeAll(func() {
+			ns := appNamespace()
+			ns.SetOwnerReferences([]metav1.OwnerReference{
+				{
+					APIVersion: "another-api-version",
+					Kind:       "another-kind",
+					Name:       "another-owner",
+					UID:        "another-uid",
+				},
+			})
+			ns.Annotations = map[string]string{
+				annotationKey: annotationValue,
+			}
+			By(fmt.Sprintf("creating namespace %s on member cluster", ns.Name))
+			Expect(memberCluster1EastProdClient.Create(ctx, &ns)).Should(Succeed(), "Failed to create namespace %s", ns.Name)
+
+			// Create the CRP.
+			strategy := &placementv1beta1.ApplyStrategy{AllowCoOwnership: true}
```

Comment:
can we also test the case it's not filled?

Reply:
we do have one test in https://github.com/Azure/fleet/blob/main/test/e2e/placement_selecting_resources_test.go#L691